app/views/assignments/_gender_request_select.html.slim

```diff
@@ -0,0 +1,11 @@
+- gender_request_filter = params[:q] && params[:q][:gender_request_cont].present? ? params[:q][:gender_request_cont] : 'all'
```

Comment:
Same here and in `_language_skills_language_select.html.slim` below, you can make a generic helper which takes the query parameter as argument